src/frontend/src/handler/create_source.rs

```diff
@@ -1076,6 +1082,7 @@ pub mod tests {
             ("address", DataType::Varchar),
             ("zipcode", DataType::Varchar),
         ])
+        // .with_ids([5, 6].map(ColumnId::new))
```

Comment:
Garbage code? IIUC the `with_ids()` should always be done by `ColumnIdGenerator`

Reply:
I leave the source field IDs blank because of subtle inconsistencies when using different paths to assign column IDs between `CREATE SOURCE` and `CREATE TABLE`. Once we address that by unifying them, the derived source columns will also have field IDs.